cicd/ReleasePipeline.jenkinsfile

```
@@ -0,0 +1,75 @@
+def appName = 'casino'
+def repositoryName = 'casino'
+
+// TODO: Change target channel for branch deployment notifications
+//       Make sure target channel has Jenkins app added to it
+def slackChannel = '#cicd-production-deployments'
+def buildLink = "<${env.RUN_DISPLAY_URL}|${env.JOB_NAME} ${env.BUILD_DISPLAY_NAME}>"
+def slackResponse = slackSend(channel: slackChannel, message: "*${repositoryName} | ${buildLink}*\nPreparing full release...", color: "#0099ff")
+
+pipeline {
+  agent { label 'dind' }
```

Use `release-helper` as agent because `dind` isn't used.
```suggestion
  agent { label 'release-helper' }
```

---

Done